tests/setup/odh-core.yaml

```
@@ -91,9 +84,9 @@ spec:
   - kustomizeConfig:
       repoRef:
         name: manifests
-        path: data-science-pipelines-operator/
-    name: data-science-pipelines-operator
+        path: trustyai-service
+    name: trustyai
   repos:
   - name: manifests
-    uri: https://github.com/opendatahub-io/odh-manifests/tarball/master
```

Typo `mater` should be `master`
CI will fail since this error blocks any manifests tarball from downloading

```suggestion
uri: https://github.com/opendatahub-io/odh-manifests/tarball/master
```